sdk/python/feast/type_map.py

```diff
@@ -252,6 +252,18 @@ def _python_value_to_proto_value(feast_value_type, value) -> ProtoValue:
                 )
             )
 
+        if feast_value_type == ValueType.UNIX_TIMESTAMP_LIST:
```

Comment:
Should the SCALAR type also be handled below?

Reply:
True.

Done